### DISCUSSION TITLE
Mark comments as resolved

### DISCUSSION THREAD

**Discussion by zimbatm** [outline/outline#5837] [Ideas]
On articles with a lot of comments, it can become difficult to see which ones have been addressed and which ones still need to be fixed.

It would be useful to be able to mark a thread as resolved, so it could be displayed differently. Either grayed out or even hidden from the default view.

**Comment by tommoor** [comment]
100% plan to build this relatively soon

**Reply by alkanna** [reply]
This alongside being able to comment on not just basic text but other blocks such as code would make Outline a very viable solution to collaborate on all kinds of documents !

**Comment by tommoor** [comment]
This functionality has now been released in https://github.com/outline/outline/releases/tag/v0.78.0
